src/main/java/org/opensearch/security/support/Base64Helper.java

```diff
@@ -177,6 +179,8 @@ private final static class SafeObjectOutputStream extends ObjectOutputStream {
 
         private final DescriptorReplacer descriptorReplacer = new DescriptorReplacer();
 
+        private boolean replaceWithOdfePackage = true;
```

Comment:
Whenever possible make this fields final

Reply:
This is set in the constructor and cannot be set to final.